core/files.py

```diff
@@ -2,19 +2,24 @@
 
 import blurhash
 import httpx
+import PIL.ImageQt
```

Comment:
Wait what is this doing here, since it's not referenced elsewhere?

Reply:
Uh, good question. I have no idea.

Reply:
I think something misfired when I typed it, and imported the reference. Anyways, fixed now.